public/app/features/alerting/unified/triage/rows/GenericRow.tsx

```diff
@@ -8,6 +8,11 @@ import { IconButton, Stack, useStyles2 } from '@grafana/ui';
 
 import { Spacer } from '../../components/Spacer';
 
+// Constants for grid layout - exported for use in other components (e.g., drawer width calculations)
+export const GRID_GAP_SPACING = 2; // theme.spacing(2) = 16px
+export const COLUMN_BORDER_WIDTH = 1; // 1px border on each column
+export const COLUMN_CONTENT_PADDING = 5; // 5px padding inside each column
+
```

Comment:
nit: I think this is no longer necessary

Reply:
truee